templates/partials/nav.html

```diff
@@ -0,0 +1,19 @@
+<nav class="navigation">
+  <div>
+    <a href="/"><img class="navigation__logo" src="/dist/images/google-go.jpg"></img></a>
+  </div>
+
+  {{ if .data.authenticatedUser }}
```

Comment:
That's where a presenter would be better to group all view related methods:

```
{{ if .data.applicationPresenter.userIsSignedIn }}
```

Now there is only one authenticated controller but if there are more than one then not having this code in the keyword controller only would be necessary.

Reply:
I found issue with the gin context. I cannot use function from the presenter in the view. So I will do this in another PR
https://github.com/gutakk/go-google-scraper/issues/46